server/webapp/WEB-INF/vm/pipeline/_pipeline_history_list_jstemplate.vm

```diff
@@ -156,7 +156,6 @@
                                 <a id="stage-detail-${% pipeline.pipelineId %}-${% stage.stageName %}-link" class="detail" title="see the details of this stage"
                                         href="$req.getContextPath()/pipelines/${% stage.stageLocator %}{if 'failed' == stage.stageStatus}#tab-failures{/if}{if 'passed' == stage.stageStatus}#tab-artifacts{/if}">
                                         Details</a>
-                              <span><span></span></span>
```

Comment:
@naveenbhaskar Was this left behind by mistake and you fixed it?

Reply:
This html structure was required for the css animation. when we removed the animation, we just removed the css, but not the html for that. Since we are using a svg img for animation, this code is not required.